### PR TITLE
feat: create menu feature

### PR DESCRIPTION
**Pull request by mthanh2209**

### CODE REVIEW

File: javascript-practice/src/javascripts/models/menu.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export default class Menu {
+	constructor() {
+		this.menuOpen = false;
+	}
+
+	toggleMenu() {
+		this.menuOpen = !this.menuOpen;
+	}
+}
```
Comment (hungnguyenhuu-agilityio) [inline javascript-practice/src/javascripts/models/menu.js:9]: Why we should define a Menu as a model?

Comment (mthanh2209) [reply javascript-practice/src/javascripts/models/menu.js]: Sorry, my mistake. I updated it.

File: javascript-practice/src/javascripts/views/layoutView.js [hunk 2]
```diff
@@ -5,6 +5,23 @@ class LayoutView {
 	constructor() {
 		this.headerEl = document.querySelector('.main-header');
 		this.footerEl = document.querySelector('.main-footer');
+
+		this.menuEl = document.querySelector('.menu-icon');
+		this.header = document.querySelector('header');
+		this.menuOpen = false;
```
Comment (hungnguyenhuu-agilityio) [inline javascript-practice/src/javascripts/views/layoutView.js:11]: naming.

Comment (mthanh2209) [reply javascript-practice/src/javascripts/views/layoutView.js:11]: I fixed it

File: javascript-practice/src/javascripts/views/layoutView.js [hunk 3]
```diff
@@ -5,6 +5,23 @@ class LayoutView {
 	constructor() {
 		this.headerEl = document.querySelector('.main-header');
 		this.footerEl = document.querySelector('.main-footer');
+
+		this.menuEl = document.querySelector('.menu-icon');
+		this.header = document.querySelector('header');
+		this.isOpen = false;
```
Comment (hungnguyenhuu-agilityio) [inline javascript-practice/src/javascripts/views/layoutView.js:11]: Which is open?

Comment (mthanh2209) [reply javascript-practice/src/javascripts/views/layoutView.js]: I fixed the name to isOpenMenu

**Comment by hungnguyenhuu-agilityio** [comment]
Conflict

**Pull request closed without merging**